website/blog/2019-12-05-oclif-eslint-migration.md

```diff
@@ -0,0 +1,71 @@
+---
+title: oclif TSLint to ESLint Migration
+---
+
+Back in February of this year, plans were announced to [deprecate TSLint](https://github.com/palantir/tslint/issues/4534) in favor of ESLint. The project’s goal is to work toward a “unified developer experience” by supporting ESLint development going forward.
+
+## Why the change
+
+In order to keep inline with the community, oclif has transitioned to ESLint for the core libraries as well as all official plugins.
+
+## What changed in oclif
+
+Starting in oclif v1.15.x, oclif will now optionally generate projects with ESLint for both Typescript and JavaScript CLI’s.
```

Comment:
Starting in v1.15.x, oclif will now optionally generate projects with ESLint for both Typescript and JavaScript CLI’s.